### PR TITLE
N10 sleep

### PR DESCRIPTION
**Pull request by levBagryansky**
@mximp please review
I implemented `threads.sleep` according to #10. It makes dataization pause for the given amount of milliseconds.
If `millis` is negative it throws exception. It is checking in `sleep-test.eo`.
I check dataization time in `EOsleepTest.java`.

### CODE REVIEW

**Comment by levBagryansky** [comment]
@mximp please look

File: src/test/eo/org/eolang/threads/sleep-test.eo [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
++alias org.eolang.hamcrest.assert-that
++alias org.eolang.threads.sleep
++architect yegor256@gmail.com
++home https://github.com/objectionary/eo
++junit
++package org.eolang.threads
++version 0.0.0
+
+[] > sleep-returns-true
+  sleep > @
```
Comment (mximp) [inline src/test/eo/org/eolang/threads/sleep-test.eo:32]: @levBagryansky can we reduce it to 200ms so that the test doesn't take too long to execute.

**Review by mximp — CHANGES_REQUESTED** [review]

File: src/test/eo/org/eolang/threads/sleep-test.eo (hunk 1 above)
Comment (levBagryansky) [reply src/test/eo/org/eolang/threads/sleep-test.eo:32]: @mximp ok

**Comment by levBagryansky** [comment]
@mximp please look
I decreased sleep time

**Review by mximp — APPROVED** [review]

**Pull request merged**